pbs/pbsrequest.go

```diff
@@ -30,20 +37,42 @@ type Bids struct {
 	Params     json.RawMessage `json:"params"`
 }
 
+type PBSVideo struct {
```

Comment:
Could we Godoc these? The conventions are described [here](https://blog.golang.org/godoc-documenting-go-code).

Some are self-explanatory... but as an outsider to video, I couldn't tell you what `Mimes` or `Frameworks` should be. Better to err on the side of too much rather than too little.